src/arm/v6.rs

```diff
@@ -3,9 +3,11 @@
 //! The reference is [ARMv6-M Architecture Reference
 //! Manual](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0419c/index.html).
 
+#[cfg(test)]
+use assert_instr::assert_instr;
+
 /// Reverse the order of the bytes.
 #[inline(always)]
-#[cfg_attr(test, assert_instr(rev))]
 pub fn _rev_u8(x: u8) -> u8 {
```

Comment:
what was the problem here ?

Reply:
Ah this function is a noop, so all it had was `blx`, a return instruction!

Reply:
Indeed, can you delete it? It was a mistake. rev is only supported for 16, 32 and 64bit, for 8bits it does not make sense :D CI saved the day!

Reply:
Sure!